src/test/generate-matrix.py

```diff
@@ -203,46 +207,44 @@ def __str__(self):
         TEST_INSTALL="t",
     ),
 )
+fedora40_platforms = deepcopy(DEFAULT_MULTIARCH_PLATFORMS)
+fedora40_platforms["linux/arm64"]["timeout_minutes"] = 240
 matrix.add_multiarch_build(
-    name="noble",
+    name="fedora40",
     default_suffix=" - test-install",
     args=common_args,
+    platforms=fedora40_platforms,
     env=dict(
         TEST_INSTALL="t",
     ),
+    docker_tag=True,
 )
+
 matrix.add_multiarch_build(
-    name="el9",
+    name="noble",
     default_suffix=" - test-install",
     args=common_args,
     env=dict(
         TEST_INSTALL="t",
     ),
 )
 matrix.add_multiarch_build(
-    name="alpine",
+    name="el9",
     default_suffix=" - test-install",
-    args=(
-        "--prefix=/usr"
-        " --sysconfdir=/etc"
-        " --with-systemdsystemunitdir=/etc/systemd/system"
-        " --localstatedir=/var"
-        " --with-flux-security"
-    ),
+    args=common_args,
     env=dict(
```

Comment:
It's mostly diff picking bad ranges, I put fedora40 back in and it mucked up the alignment.  I also used deepcopy to fix an issue with sub-dicts being updated by other tests.  This is good in that we're testing what we think we are, and bad in that now noble is failing because it's actually testing the installed version, which somehow breaks the mpich bootstrap again.  Looking into it.

Reply:
Great! Already approved so feel free to set MWP!